examples/hosts/app-integration/external-data/src/view/taskListView.tsx

```diff
@@ -124,6 +129,9 @@ const TaskRow: React.FC<ITaskRowProps> = (props: ITaskRowProps) => {
 
 /**
  * {@link TaskListView} input props.
+ * claimLeadership(): function to update container's current leader
```

Comment:
Let's move these inline like in my comment above (I can't make a github suggestion here).